docs/reference/reference.md

```diff
@@ -303,3 +303,69 @@ Selected codelists are displayed below.
       :file: ../../build/current_lang/codelists/modificationType.csv
 
 ```
+
```

Comment:
Is there a reason for the current order? Alphabetical order would be more usable.

Reply:
fixed in https://github.com/open-contracting/infrastructure/pull/101/commits/ee970d2ec9d7ac211a306e225316d3c8d82822f3

I also split the codelists out into their own reference page, categorised them into open and closed codelists and added some introductory text.